cli/fabric.c

```diff
@@ -37,9 +37,19 @@
 #include <errno.h>
 #include <ctype.h>
 
+static const char * const cmd_desc_topo_info = "Show topology information";
+static const char * const cmd_desc_gfms_bind = "Bind the EP (function) to the specified host";
+static const char * const cmd_desc_gfms_unbind = "Unbind the EP (function) from the specified host";
+static const char * const cmd_desc_gfms_dump = "Dump the GFMS database";
+static const char * const cmd_desc_route = "Show routing information";
+static const char * const cmd_desc_port_control = "Control a port";
+static const char * const cmd_desc_portcfg_show = "Display a port's configuration information";
+static const char * const cmd_desc_portcfg_set = "Configure a port";
+static const char * const cmd_desc_gfms_events = "Display GFMS event information";
+
 static int gfms_bind(int argc, char **argv)
 {
-	const char *desc = "Unbind the EP(function) to the specified host";
+	const char *desc = cmd_desc_gfms_bind;
```

Comment:
If you want to move the description outside the function (but still right above it) that's fine. But ]then you wouldn't need the 'desc' variable, just use the 'cmd_desc_x' variable directly.

Reply:
Good point, I'll remove all the unnecessary 'desc' variables. See my question above about moving the description.